tfhe/benches/integer/bench.rs

```diff
@@ -2514,10 +2512,10 @@ fn go_through_gpu_bench_groups(val: &str) {
         "fast_default" => {
             default_cuda_dedup_ops();
         }
-        "unchecked" => {
-            unchecked_cuda_ops();
-            unchecked_scalar_cuda_ops()
-        }
+        //"unchecked" => {
+        //    unchecked_cuda_ops();
+        //    unchecked_scalar_cuda_ops()
+        //}
```

Comment:
why do you remove them btw ? we only run them if we ask for them right ?

Reply:
Oh yes I'm stupid :facepalm: I'll remove them from the workflow files instead.